ch_1_assign_Jonathan_Ho.py

```diff
@@ -0,0 +1,153 @@
+Robot_Team = input("What Robotics Team do you want to look into?")
```

Comment:
You need to ask for what piece of information from the team the user is looking for as well as which team

Reply:
Whoops! I heard from another Citrus Member that all you needed was the team info as a whole, not as an individual. I'll see to it this is fixed